Document/0x05b-Basic-Security_Testing.md

```diff
@@ -105,9 +105,30 @@ An extensive list of root detection methods is presented in the "Testing Anti-Re
 
 For a typical mobile app security build, you'll usually want to test a debug build with root detection disabled. If such a build is not available for testing, you can disable root detection in a variety of ways that will be introduced later in this book.
 
-#### Recommended Tools
-
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1227>
+#### Recommended Tools - Android device
+
+There are many tools and frameworks used throughout this guide to assess the security of Android applications. In the next sections you will learn more about some of their main commands and interesting use cases. Please check the official documentation for installation instructions of the following tools/APKs:
+
+- APK Extractor: App to extract APKs without root.
+- Frida server: Server for Frida, the dynamic instrumentation toolkit for developers, reverse-engineers, and security researchers.
```

Comment:
Frida is actually introduced below so maybe just refer to it from here (after removing the list below). Something like: _See "Frida" section below for more information._

Reply:
Almost all the tools are introduced below. Sven asked me to mimic the other chapter and repeat the content as they are separate sections after all.